app/controllers/decidim_controller.rb

```diff
@@ -1,3 +1,10 @@
 # frozen_string_literal: true
 class DecidimController < ActionController::Base
+  before_action :set_raven_context
+
+  def set_raven_context
+    return unless Rails.application.secrets.sentry_enabled?
+    Raven.user_context({ id: session[:current_user_id] }.merge(session))
```

Comment:
Will I have current_user when nobody is logged in? Maybe `try(:current_user).try(:id)`